src/NuGet.Core/NuGet.Commands/CommandArgs/PackArgs.cs

```diff
@@ -31,6 +31,7 @@ public class PackArgs
         public bool Symbols { get; set; }
         public bool Tool { get; set; }
         public string Version { get; set; }
+        public bool AllowPrereleaseDependenciesInProductionPackage { get; set; }
```

Comment:
how about ``AllowPrereleaseDependenciesInStablePackage``?

I'm not sure Production is the opposite of pre-release.